python_modules/dagster-graphql/dagster_graphql/implementation/fetch_runs.py

```diff
@@ -76,14 +68,14 @@ def get_run_tags(graphene_info):
 
 
 @capture_error
-def get_run_group(graphene_info, run_id):
+def get_run_group(graphene_info: "HasContext", run_id: str) -> "GrapheneRunGroup":
     from ..schema.errors import GrapheneRunGroupNotFoundError
     from ..schema.pipelines.pipeline import GrapheneRun
     from ..schema.runs import GrapheneRunGroup
 
     instance = graphene_info.context.instance
     try:
-        result = instance.get_run_group(run_id)
+        result = check.not_none(instance.get_run_group(run_id))
```

Comment:
I'm slightly wary of this. At a minimum please put some explanatory text in the check error message. At this this way a user who encounters this and gets a client-side error will report back a helpful error message. Do you know definitively what is going on underneath the hood in `get_run_group` across OSS and Cloud?

Reply:
Like the below one you commented on, this codepath breaks if `result` is `None`, because two lines down is this:

```
root_run_id, run_group = result
```